cypress/integration/pages/companies/create_spec.page.js

```diff
@@ -0,0 +1,41 @@
+describe('Companies', () => {
+  const admin = 'admin@example.com', password = '123456', companyCNPJ = Math.random().toString().slice(2,16)
+
+  beforeEach(() => {
+    cy.login(admin, password)
+
+    cy.findAllByText('Empresas').click()
+    cy.url().should('include', '/companies')
+  })
+
+  it('successfully creates a new company', () => {
+    cy.findAllByText('Cadastrar Empresa').click()
+    cy.findAllByText('Nova empresa').should('exist')
+
+    cy.get('[data-testid="company-name-input"]').type('Company Test')
```

Comment:
You don't need to set a `data-testid` for every single input. Just select them by name. https://stackoverflow.com/questions/55716154/in-cypress-how-to-select-input-element-based-on-name

Also you can either create a custom command for that or use some getter from [here](https://testing-library.com/docs/cypress-testing-library/intro/).

Reply:
Good one, I was using some because was already there, I'll change that